### ISSUE TITLE
Validation dataset are need ?

### ISSUE THREAD

**Issue by sksksk1748** [j1a0m0e4sNTU/3dgcn#8]
Hello @j1a0m0e4sNTU ,
I successfully executed this program, but I have some questions about the allocation of datasets.
Under normal circumstances, the training dataset and the validation dataset are carried out together.
And in your program it seems that the test dataset is used for training and also for testing.

```python
# classification/main.py
# Line 55~72
if args.mode == "train":
    print('Trianing ...')
    train_data = ModelNet_pointcloud(args.dataset, 'train', transform= transform)
    train_loader = DataLoader(train_data, shuffle= True, batch_size= args.bs)
    test_data = ModelNet_pointcloud(args.dataset, 'test', transform= transform)
    test_loader = DataLoader(test_data, shuffle= False, batch_size= args.bs)

    # I have a problem here
    manager.train(train_loader, test_loader)

else:
    print('Testing ...')
    test_data = ModelNet_pointcloud(args.dataset, 'test', transform= transform)
    test_loader = DataLoader(test_data, shuffle= False, batch_size= args.bs)
    
    # I have a problem here 
    test_loss, test_acc = manager.test(test_loader)

    print('Test Loss: {:.5f}'.format(test_loss))
    print('Test Acc:  {:.5f}'.format(test_acc))
```

**Comment by peterjohnsonhuang** [comment]
Hi,
Thank you for supporting our work. To have a fair comparison with other methods, we adopt the same data split and training setting with other works. Although we know that a validation set should differ from the testing set, all the previous works use testing split as the validation set during training. Note that the model **did not** use the testing set for updating parameters.
